bHY6yh-d35945/cod-pu6y7d_bHY6yh-d35945.xml

```diff
@@ -61,321 +61,323 @@
     </p>
             <p xml:id="bHY6yh-d35945-d1e119">
                <lb ed="#V" n="81"/>
-               <quote xml:id="bHY6yh-d35945-Qd1e123">¶ Psalmista. 46 psalmus. Iustus miseretur et
-        <lb ed="#V" n="82"/>tribuet. </quote>
+             ¶ <ref>Psalmista. 46 psalmus.</ref> <quote xml:id="bHY6yh-d35945-Qd1e123">Iustus miseretur et
+        <lb ed="#V" n="82"/>tribuet.</quote>
             </p>
             <p xml:id="bHY6yh-d35945-d1e129">
                <lb ed="#V" n="83"/>
-               <quote xml:id="bHY6yh-d35945-Qd1e133">¶ Salomon in prouerb. ca. 12. Qui negligit
+             ¶ <ref>Salomon in prouerb. ca. 12.</ref> <quote xml:id="bHY6yh-d35945-Qd1e133">Qui negligit
         <lb ed="#V" n="84"/>damnum propter amicum: iustus est.
-        <lb ed="#V" n="85"/>Multi colunt personam potentis. </quote>
+        <lb ed="#V" n="85"/>Multi colunt personam potentis.</quote>
             </p>
             <p xml:id="bHY6yh-d35945-d1e142">
                <lb ed="#V" n="86"/>
-               <quote xml:id="bHY6yh-d35945-Qd1e146">¶ Cronica de nugis philosophurum. c. 7 scilicet de cleobo
-        <lb ed="#V" n="87" break="no"/>lo. Cleobolus vnus de septem sapieinentibus dice
+             ¶ <ref>Cronica de nugis philosophurum. c. 7 scilicet de cleobo
+        <lb ed="#V" n="87" break="no"/>lo.</ref> <quote xml:id="bHY6yh-d35945-Qd1e146">Cleobolus vnus de septem sapientibus dice
         <lb ed="#V" n="88" break="no"/>bat beneficiendum esse amico vt amicicior fi
-        <lb ed="#V" n="89"/>at. </quote>
+        <lb ed="#V" n="89" break="no"/>at.</quote>
             </p>
             <p xml:id="bHY6yh-d35945-d1e156"> /
-      <quote xml:id="bHY6yh-d35945-Qd1e158"> 28. c scilicet de aristotele. Aristoteles ad cul
-        <lb ed="#V" n="90"/>pantem: quia improbo contribuerat: ait non con <!--121.xml-->
+            <ref>28. c scilicet de aristotele.</ref> <quote xml:id="bHY6yh-d35945-Qd1e158">Aristoteles ad cul
+        <lb ed="#V" n="90" break="no"/>pantem: quia improbo contribuerat: ait non con <!--121.xml-->
         <pb ed="#V" n="47-v"/>
                   <cb ed="#V" n="a"/>
-                  <lb ed="#V" n="1"/>tribui homin: sed humanitati. </quote>
+                  <lb ed="#V" n="1" break="no"/>tribui homini: sed humanitati.</quote>
             </p>
             <p xml:id="bHY6yh-d35945-d1e172"> /
-      <quote xml:id="bHY6yh-d35945-Qd1e174"> Idem interro
-        <lb ed="#V" n="2" break="no"/>gatus quiliter sit nos erga caros gerendum: ait
-        <lb ed="#V" n="3"/>put amamus eos erga nos se gerere. </quote>
+      <quote xml:id="bHY6yh-d35945-Qd1e174"> <ref>Idem interro
```

Comment:
looks like we've got an issue here. There is opening `<ref>` tag but no closing tag.

This is a difficult case, but for now you could encode it as <ref>Idem</ref> <quote>interrogatus...</quote.